main.go

```diff
@@ -3,11 +3,35 @@ package main
 import (
 	"github.com/guilhermef/go-shortener/config"
 	"github.com/guilhermef/go-shortener/handler"
+  "os"
+  "fmt"
 	"log"
 	"net/http"
+
+  newrelic "github.com/newrelic/go-agent"
+)
+
+var (
+	app newrelic.Application
```

Comment:
This is a `package level vars` only accessible on main package.

There is a good [post](http://peter.bourgon.org/blog/2017/06/09/theory-of-modern-go.html) about it.

Do you really does it ?